### PR TITLE
Code over config

### PR DESCRIPTION
**Pull request by dvdstelt**
Connected to https://github.com/Particular/NServiceBus.AzureStorageQueues/issues/24

Removed configsection & appsetting "transport/nservicebus"
Everything is now code-only.

### CODE REVIEW

File: src/Transport/Config/AzureStorageTransportExtensions.cs [hunk 1]
```diff
@@ -30,7 +46,7 @@ public static TransportExtensions<AzureStorageQueueTransport> PeekInterval(this
         /// <returns></returns>
         public static TransportExtensions<AzureStorageQueueTransport> MaximumWaitTimeWhenIdle(this TransportExtensions<AzureStorageQueueTransport> config, int value)
         {
-            config.GetSettings().Set(ReceiverMaximumWaitTimeWhenIdle, value);
+            config.GetSettings().Set(WellKnownConfigurationKeys.ReceiverMaximumWaitTimeWhenIdle, value);
```
Comment (SeanFeldman) [inline src/Transport/Config/AzureStorageTransportExtensions.cs]: The old code was not necessarily guarding against invalid parameter values, but should we?

File: src/Transport/Config/AzureStorageTransportExtensions.cs [hunk 2]
```diff
@@ -65,7 +81,7 @@ public static TransportExtensions<AzureStorageQueueTransport> MessageInvisibleTi
         /// <returns></returns>
         public static TransportExtensions<AzureStorageQueueTransport> BatchSize(this TransportExtensions<AzureStorageQueueTransport> config, int value)
         {
-            config.GetSettings().Set(ReceiverBatchSize, value);
+            config.GetSettings().Set(WellKnownConfigurationKeys.ReceiverBatchSize, value);
```
Comment (SeanFeldman) [inline src/Transport/Config/AzureStorageTransportExtensions.cs]: Same as in https://github.com/Particular/NServiceBus.AzureStorageQueues/pull/48/files#r54795389

File: src/Transport/Config/AzureStorageTransportExtensions.cs [hunk 3]
```diff
@@ -96,46 +112,21 @@ public static TransportExtensions<AzureStorageQueueTransport> SerializeMessageWr
         /// </summary>
         public static TransportExtensions<AzureStorageQueueTransport> CreateSendingQueues(this TransportExtensions<AzureStorageQueueTransport> config)
         {
-            config.GetSettings().Set(TransportCreateSendingQueues, true);
+            config.GetSettings().Set(WellKnownConfigurationKeys.TransportCreateSendingQueues, true);
```
Comment (SeanFeldman) [inline src/Transport/Config/AzureStorageTransportExtensions.cs]: Should we let users specify via parameter if creation should take place or not? We can default to `true` if we want, but allow to pass in `false`.

File: src/Transport/Config/AzureStorageTransportExtensions.cs [hunk 4]
```diff
@@ -96,46 +112,21 @@ public static TransportExtensions<AzureStorageQueueTransport> SerializeMessageWr
...
             return config;
         }
 
         public static TransportExtensions<AzureStorageQueueTransport> UseSha1ForShortening(this TransportExtensions<AzureStorageQueueTransport> config)
         {
-            config.GetSettings().Set(Sha1Shortener, true);
+            config.GetSettings().Set(WellKnownConfigurationKeys.Sha1Shortener, true);
```
Comment (SeanFeldman) [inline src/Transport/Config/AzureStorageTransportExtensions.cs]: Same as [here](https://github.com/Particular/NServiceBus.AzureStorageQueues/pull/48/files#r54796221)

File: src/Transport/Config/AzureStorageTransportExtensions.cs (hunk 3 above)
Comment (dvdstelt) [inline src/Transport/Config/AzureStorageTransportExtensions.cs]: Do you mean as an overload (or default value), providing both "usethis()" and "usethis(false)" options?

Comment (SeanFeldman) [inline src/Transport/Config/AzureStorageTransportExtensions.cs]: @dvdstelt now that I look at this, this method should be gone. Decision was made that destination queues should not be created by the sender. As such, this becomes obsolete. But for other methods like this, yes, your comment is correct.

Comment (dvdstelt) [inline src/Transport/Config/AzureStorageTransportExtensions.cs]: Even better, removing it. I'll make it so.

**Comment by dvdstelt** [comment]
I'm updating to NSB6 1616 where some stuff was changed, Initialize method added, etc.

Because when updating TransPortCompat tests to latest NSB version, it was complaining about that Initialize method and more. And changes need to be done anyhow.

**Pull request merged**